src/errors/runtime/templates.js

```diff
@@ -111,5 +111,10 @@ export default {
     [RUNTIME_ERRORS.uncaughtErrorInReporter]:           'An uncaught error occurred in the "{reporterName}" reporter\'s "{methodName}" method. Error details:\n{originalError}',
     [RUNTIME_ERRORS.roleInitializedWithRelativeUrl]:    'You cannot specify relative login page URLs in the Role constructor. Use an absolute URL.',
     [RUNTIME_ERRORS.typeScriptCompilerLoadingError]:    'Unable to load the TypeScript compiler.\n{originErrorMessage}.',
-    [RUNTIME_ERRORS.cannotCustomizeSpecifiedCompilers]: 'You cannot specify options for the {noncustomizableCompilerList} compiler{suffix}.'
+    [RUNTIME_ERRORS.cannotCustomizeSpecifiedCompilers]: 'You cannot specify options for the {noncustomizableCompilerList} compiler{suffix}.',
+    [RUNTIME_ERRORS.browserConnectionError]:            '{originalErr}\n{notOpenedConnectionsNum} of {allConnectionsNum} browser connections have not been established:{notOpenedConnectionsList}\n{hintsList}',
+    [CONNECTION_ERROR_HINTS.toHighConcurrencyFactor]:   'The error can be due to a concurrency factor that is too high for the host machine’s performance (the factor value {concurrencyFactor} was specified). ' +
+                                                        'Try to decrease the concurrency factor or ensure more system resources are available on the host machine.',
+    [CONNECTION_ERROR_HINTS.useBrowserInitOption]: 'Use the \'--browser-init-timeout\' option to allow more time for the browser to start.',
```

Comment:
We do not print the current value in this message. IMHO it makes it harder to understand what is "MORE time" in this context.